docs/ru_RU/index.html

```diff
@@ -953,7 +953,7 @@ <h2>Периодические транзакции</h2>
     <h2>Инвестиции</h2>
     MMEX allows you to track Stocks and Shares. The total Stocks are added to
     your total financial worth.<br>Further details refer to
-    <a href="../stocks_and_shares.html?lang=ru">Stocks, Shares and Share Dividend Management in MMEX</a>
+    <a href="../stocks_and_shares.html?lang=en_GB">Stocks, Shares and Share Dividend Management in MMEX</a>
```

Comment:
Unless it never existed, should this be `ru_RU`? Thank you

Reply:
There is no stocks_and_shares.html for the Russian language, so I sent it back to en_GB. However, I think it would be better to just copy the en_GB file to the ru_RU directory then have the links point to files within the directory. The stocks_and_shares file can be translated into Russian later by other users.

Reply:
Done